packages/harmony/src/storybook/components/Subtitle.tsx

```diff
@@ -0,0 +1,6 @@
+import { styled } from '@storybook/theming'
+
+export const Subtitle = styled.span(({ theme }) => {
+  const css = { fontSize: `${theme.typography.size.m2}px !important` }
+  return { ...css, p: css }
```

Comment:
does `p` propagate to <p> tag children?

Reply:
This is a super weird situation, basically. When you wrap this component around markdown, it adds a p tag inside. Weirdly though this only happens when the mark is long enough to make it a new line inside the component. if it's inline there is no inner tag. So what this is saying is style both the case where we wrap an extra p tag and when we don't.